### PR TITLE
feat: rename favorite to bookmark

### LINKED ISSUE #799: Rename favorites to bookmarks

### PR DESCRIPTION
**Pull request by pyphilia**
close #799

### CODE REVIEW

File: src/components/common/BookmarkButton.tsx [hunk 1]
```diff
@@ -21,48 +21,48 @@ type Props = {
 
 const isItemFavorite = (
```
Comment (spaenleh) [inline src/components/common/BookmarkButton.tsx:22]:
```suggestion
const isItemBookmark = (
```

File: src/components/main/ItemCard.tsx [hunk 2]
```diff
@@ -21,8 +21,8 @@ import { hooks } from '../../config/queryClient';
 import { buildItemCard, buildItemLink } from '../../config/selectors';
 import defaultImage from '../../resources/avatar.png';
 import { stripHtml } from '../../utils/item';
+import FavoriteButton from '../common/BookmarkButton';
```
Comment (spaenleh) [inline src/components/main/ItemCard.tsx:24]:
```suggestion
import BookmarkButton from '../common/BookmarkButton';
```

File: src/components/main/ItemMenu.tsx [hunk 3]
```diff
@@ -26,8 +26,8 @@ import {
   buildItemMenuButtonId,
 } from '../../config/selectors';
 import { BUILDER } from '../../langs/constants';
+import FavoriteButton from '../common/BookmarkButton';
```
Comment (spaenleh) [inline src/components/main/ItemMenu.tsx:29]:
```suggestion
import BookmarkButton from '../common/BookmarkButton';
```

File: src/langs/en.json [hunk 4]
```diff
@@ -191,7 +191,7 @@
   "MOVE_ITEM_MODAL_TITLE_one": "Where do you want to move {{name}} and one other?",
   "MOVE_ITEM_MODAL_TITLE_other": "Where do you want to move {{name}} and {{count}} others?",
   "MY_ITEMS_TITLE": "My Graasp",
-  "NAVIGATION_FAVORITE_ITEMS_TITLE": "Favorite Items",
+  "NAVIGATION_BOOKMARKED_ITEMS_TITLE": "Favorite Items",
```
Comment (spaenleh) [inline src/langs/en.json:194]: Update translation ?

File: src/langs/fr.json [hunk 5]
```diff
@@ -180,7 +180,7 @@
   "MOVE_ITEM_MODAL_TITLE_two": "Où déplacer {{name}} et un autre ?",
   "MOVE_ITEM_MODAL_TITLE_other": "Où déplacer {{name}} et {{count}} autres éléments ?",
   "MY_ITEMS_TITLE": "Mon Graasp",
-  "NAVIGATION_FAVORITE_ITEMS_TITLE": "Éléments Favoris",
+  "NAVIGATION_BOOKMARKED_ITEMS_TITLE": "Éléments Favoris",
```
Comment (spaenleh) [inline src/langs/fr.json:183]: Update ?

**Review by spaenleh — APPROVED** [review]
I think this looks good, I am just a bit worried about the change of the route. 
There are some translations that have not been changed. We need to ensure that everything is coherent. 
Are we planning to also change the icon ? Maybe this can be done later ?

**Comment by pyphilia** [comment]
I've already changed the icons, done in graasp-ui and some places in builder.

**Review by spaenleh — APPROVED** [review]

**Pull request merged**